### PR TITLE
Initial VS project creation

### PR DESCRIPTION
**Pull request by mkuehn10**

### CODE REVIEW

**Comment by mkuehn10** [comment]
@k2zinger

File: README.md
Comment (k2zinger) [inline README.md:1]: # [UiPathTeam.Extensions.Activities](https://marketplace.uipath.com/listings/uipathteam-extensions-activities)

A collection of extension methods, tasks, and activities

## Installation
Studio -> Manage Packages -> Community -> (Search) UiPathTeam.Extensions.Activities -> pick the package(s) to install and click Install -> Save

## Activities
### Statements
`Test`: A sample activity for testing

**Review by k2zinger — CHANGES_REQUESTED** [review]
Can you review these items and resubmit please?

File: README.md
Comment (mkuehn10) [reply README.md:1]: I've added this to the README.md

File: UiPathTeam.Extensions.Tests/packages.config
[the remark this answers is not in the document]
Comment (mkuehn10) [reply UiPathTeam.Extensions.Tests/packages.config:1]: I think this was an artifact from a previous attempt at incorporating Unit Testing.  I don't think I need those packages for the unit testing to work.

File: .nuget/NuGet.targets
[the remark this answers is not in the document]
Comment (mkuehn10) [reply .nuget/NuGet.targets:1]: Removed

File: .nuget/NuGet.exe
[the remark this answers is not in the document]
Comment (mkuehn10) [reply .nuget/NuGet.exe:1]: Removed

File: .nuget/NuGet.Config
[the remark this answers is not in the document]
Comment (mkuehn10) [reply .nuget/NuGet.Config:1]: Removed

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/UiPathTeam.Extensions.Activities.csproj [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+		<Reference Include="System.Xml.Linq" />
+		<Reference Include="System.Data.DataSetExtensions" />
+		<Reference Include="Microsoft.CSharp" />
+		<Reference Include="System.Data" />
+		<Reference Include="System.Net.Http" />
+		<Reference Include="System.Xml" />
+	</ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+    <PackageReference Include="System.Activities.Metadata" Version="6.0.0-*" PrivateAssets="All" />
+    <PackageReference Include="UiPath.Workflow.Runtime" Version="6.0.0-20220909-01" PrivateAssets="All"/>
```
[the remark this answers is not in the document]
Comment (mkuehn10) [reply UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/UiPathTeam.Extensions.Activities.csproj]: Changed to 

```
<ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
	  <PackageReference Include="UiPath.Workflow" Version="6.0.3" PrivateAssets="All" />
	  <PackageReference Include="UiPath.Workflow.Runtime" Version="6.0.3" PrivateAssets="All" />
	  
  </ItemGroup>
```

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/UiPathTeam.Extensions.Activities.csproj [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		<Reference Include="System.Core" />
+		<Reference Include="System.Xml.Linq" />
+		<Reference Include="System.Data.DataSetExtensions" />
+		<Reference Include="Microsoft.CSharp" />
+		<Reference Include="System.Data" />
+		<Reference Include="System.Net.Http" />
+		<Reference Include="System.Xml" />
+	</ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+    <PackageReference Include="System.Activities.Metadata" Version="6.0.0-*" PrivateAssets="All" />
```
[the remark this answers is not in the document]
Comment (mkuehn10) [reply UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/UiPathTeam.Extensions.Activities.csproj]: Changed to 

```
<ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
	  <PackageReference Include="UiPath.Workflow" Version="6.0.3" PrivateAssets="All" />
	  <PackageReference Include="UiPath.Workflow.Runtime" Version="6.0.3" PrivateAssets="All" />
	  
  </ItemGroup>
```

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities.Design/UiPathTeam.Extensions.Activities.Design.csproj [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+		<Reference Include="System.Xml.Linq" />
+		<Reference Include="System.Data.DataSetExtensions" />
+		<Reference Include="Microsoft.CSharp" />
+		<Reference Include="System.Data" />
+		<Reference Include="System.Net.Http" />
+		<Reference Include="System.Xml" />
+	</ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0-windows' ">
+    <PackageReference Include="UiPath.Workflow" Version="6.0.0-20220909-01" PrivateAssets="All" />
+    <PackageReference Include="System.Activities.Core.Presentation" Version="6.0.0-*" PrivateAssets="All" />
```
[the remark this answers is not in the document]
Comment (mkuehn10) [reply UiPathTeam.Extensions/UiPathTeam.Extensions.Activities.Design/UiPathTeam.Extensions.Activities.Design.csproj]: Changed to

```
 <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0-windows' ">
    <PackageReference Include="UiPath.Workflow" Version="6.0.3" PrivateAssets="All" />
	  <PackageReference Include="UiPath.Workflow.Runtime" Version="6.0.3" PrivateAssets="All" />
	  <PackageReference Include="System.Activities.Core.Presentation" Version="6.0.0-*" PrivateAssets="All" />
	  <PackageReference Include="System.Activities.Metadata" Version="6.0.0-*" PrivateAssets="All" />

  </ItemGroup>
```

**Review by k2zinger — APPROVED** [review]

**Pull request merged**